pages/multi-token-item/[...id].tsx

```diff
@@ -161,16 +153,18 @@ const MultiTokenItem = () => {
     },
   ]
 
-  // TODO: use name from metadata
-  const title = `${t('multi-token-collection')} ${metadata?.name ?? info?.collection?.name ?? '-'}`
+  const { metadata = {}, collection } = info as CollectionInfo
```

Comment:
I think `info` is the type of CollectionInfo. Why should use as?
`useQuery`'s result of data type is infer by `queryFn`.

Reply:
> I think `info` is the type of CollectionInfo. Why should use as? `useQuery`'s result of data type is infer by `queryFn`.

Changed.

Reply:
<img width="1088" alt="image" src="https://user-images.githubusercontent.com/22511289/201275493-03e7b7b8-1076-429b-8a83-d86a5c8398cb.png">

@yanguoyu Cause the `info` maybe got `undefined`, I give a default value to it. When I deconstruct the `metadata` from this value, I get a type error. So I gave a type assertion